docs/notebooks/dev_guide/grid.ipynb

```diff
@@ -0,0 +1,1225 @@
+{
```

Comment:
<p>Shouldn't the rho integrals should be $\int_0^1 \rho d\rho$?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PlasmaControl/DESC/pull/409/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
Do you mean that in the sense that $\rho$ is the area Jacobian for basic polar geometries, with $dA = \rho d\rho d\theta$?

If we wanted to compute the actual area of a $\zeta$ surface, we would weight it by the area Jacobian for that surface in our geometry: $\int_0^1 \int_0^{2\pi} \lvert \ e_{\rho} \times e_{\theta} \rvert d\rho d\theta$

When I mention "node area" in this document I am just referring to the product of the differential elements in the columns of `grid.spacing` for the row associated with that node. For a $\zeta$ surface the unweighted area, which is the sum of these products over all the nodes on the surface, would be $\int_0^1 \int_0^{2\pi} d\rho d\theta = 2\pi$.

I think that is an invariant the grid should try to keep. That way when we actually supply a Jacobian factor in the integral, whether that be for an area or volume, we know that the integral covers the entire domain.

Reply:
Yeah I was thinking of it like the area in the computational domain which in this case is a disk, so the area should be pi. But I can see your point as well about keeping the Jacobian separate. I'm fine with it if everyone else is.